test/exclude-members.test.js

```diff
@@ -780,10 +788,10 @@ test('On exclusion, if we fail to re-add all people, someone else does that inst
 
 test('On exclusion, recover if we fail to re-add anyone at all', async (t) => {
   const run = Run(t)
-  const alice = Testbot({ name: 'alice', timeoutScale: 0 })
+  const alice = Testbot({ name: 'alice', ...lowTimeouts })
```

Comment:
:hot_pepper: - IMO use the API that ssb-tribes uses, so people reading test code don't get confused by API

```suggestion
  const alice = Testbot({ name: 'alice', tribes2: lowTimeouts })
```

Reply:
Feel free to skip this change. It's just a recommended pattern